testlib/src/anker_context.rs

```diff
@@ -213,6 +306,79 @@ impl Context {
         BLamports(account_info.amount)
     }
 
+    /// Swap StSol for UST
+    pub async fn swap_st_sol_for_ust(
+        &mut self,
+        source: &Pubkey,
+        destination: &Pubkey,
+        authority: &Keypair,
+        amount_in: u64,
+        minimum_amount_out: u64,
+    ) {
+        let swap_instruction = spl_token_swap::instruction::swap(
+            &spl_token_swap::id(),
+            &spl_token::id(),
+            &self.token_pool_context.swap_account.pubkey(),
+            &self.token_pool_context.get_token_pool_authority().0,
+            &authority.pubkey(),
+            source,
+            &self.token_pool_context.st_sol_address,
+            &self.token_pool_context.ust_address,
+            destination,
+            &self.token_pool_context.mint_address,
+            &self.token_pool_context.fee_address,
+            None,
+            Swap {
+                amount_in,
+                minimum_amount_out,
+            },
+        )
+        .expect("Could not create swap instruction.");
+        send_transaction(
+            &mut self.solido_context.context,
+            &mut self.solido_context.nonce,
+            &[swap_instruction],
+            vec![authority],
+        )
+        .await
+        .expect("Failed to swap StSol for UST tokens.");
+    }
+
+    pub async fn sell_rewards(&mut self) {
```

Comment:
We should also have a `try_sell_rewards`, so we can try some invalid calls (e.g. passing in the wrong address that receives the proceeds) and confirm that they fail. But we can do that in the test PR, this one is large enough already.